src/components/panel/demoBasicUsage/script.js

```diff
@@ -32,10 +29,12 @@ BasicDemoCtrl.prototype.showDialog = function() {
 };
 
 
-BasicDemoCtrl.prototype.closeDialog = function() {
-  this._panelRef && this._panelRef.close();
-};
+// Necessary to pass locals to the dialog template.
+function PanelDialogCtrl(mdPanelRef) {
```

Comment:
I strongly suggest a different name for `mdPanelRef`.

*  This is a reference to some instance ?
*  What type of instance does this reference ?
*  `Ref` seems very cludgy